migration_original/ODS1Stage/functions/Utils/generate_uuid.sql

```diff
@@ -0,0 +1,13 @@
+CREATE OR REPLACE FUNCTION ODS1_STAGE_TEAM.UTILS.generate_uuid(name VARCHAR(16777216) DEFAULT NULL)
+RETURNS VARCHAR(16777216)
+LANGUAGE SQL
+AS
+$$
+CASE
+    WHEN name IS NOT NULL THEN
+        UUID_STRING('fe971b24-9572-4005-b22f-351e9c09274e', name)
```

Comment:
maybe add a comment explaining why you used this id and the fact that is random and the purpose of making the id generation stable

Reply:
I added comments explaining the function